rudder-web/src/main/webapp/templates-hidden/common-layout.html

```diff
@@ -56,55 +56,81 @@
     <script type="text/javascript" src="/javascript/rudder/angular/complianceMode.js" id="angular-complianceMode"></script>
     <script type="text/javascript" src="/javascript/rudder/angular/agentSchedule.js" id="angular-agentSchedule"></script>
 
+	<link type="text/css" rel="stylesheet" href="/style/newTheme/AdminLTE.min.css">
+	<link type="text/css" rel="stylesheet" href="/style/newTheme/menu.css">
```

Comment:
I would be happier if that file was in /style/rudder/newTheme.css 

Since this file is really Rudder specific and not only about the menu :)